src/features/playground/shareLinks/ShareLinkState.ts

```diff
@@ -0,0 +1,13 @@
+type ShareLinkState = Partial<{
```

Comment:
Let's type this strongly... If not it defeats the purpose of having the encoder/ decoder in the first place